templates/letsencrypt/4/docker-compose.yml.tpl

```diff
@@ -59,4 +59,7 @@ volumes:
       {{.Values.STORAGE_DRIVER_OPT}}
     {{- end }}
     {{- end }}
+    {{- if .Values.ENVIRONMENT_SCOPED}}
+    external: true
+    {{- end }}
```

Comment:
Your 'if' statement is not correct. Due to variable is a boolean it always enter into the if. Have you tested it??

Also, like it's specified at https://docs.docker.com/compose/compose-file/compose-file-v2/#external, `external cannot be used in conjunction with other volume configuration keys (driver, driver_opts).`

```
-    {{- end }}
+    {{- else }}
-    {{- if .Values.ENVIRONMENT_SCOPED}}
+    {{- if eq .Values.ENVIRONMENT_SCOPED "true" }}
    external: true
    {{- end }}
+    {{- end }}
```